Makefile.simple

```diff
@@ -162,7 +162,7 @@ savepng$(OBJ_EXTENSION): savepng.cpp
 	$(CXX) -O2 $(CXXFLAGS) -c savepng.cpp -o $@
 
 mymake$(EXE_EXTENSION): mymake.cpp
-	$(CXX) -O2 $(CXXFLAGS) mymake.cpp -o $@
+	$(CXX) -O2 $(CXXFLAGS) mymake.cpp -lpthread -o $@
```

Comment:
Nit: You should use the compiler option `-pthread` instead of `-lpthread` here; `-pthread` adds some extra macro definitions such as `_REENTRANT` which _might_ matter to some library code.

Reply:
Sure thing. One question, though: it's `-lpthread` everywhere else (in `Makefile.simple`, as well as in `opts` in `mymake.cpp`) -- should it be fixed there as well, or is there a specific reason for that?

Reply:
Oh yeah, I would indeed advise changing it to `-pthread` everywhere (and moving it into `CXXFLAGS_EARLY` in Makefile.simple, so that it's seen by the compiler as well as the linker). That might warrant a separate patch.

Reply:
Looked up GCC's man-page, and indeed it states re `-pthread`:
> You should use this option consistently for both compilation and linking.